### ISSUE TITLE
Usages of _HiddenModule's Width procedure don't resolve

### ISSUE THREAD

**Issue by ThunderFrame** [rubberduck-vba/Rubberduck#2835]
The RD context bar doesn't recognize the `Width` procedure in a usage like

```vb
Width #i, j
```

I suspect there's a parser bug because `Width`'s first argument is a file number that must be prefixed with a `#`.

Linking #1487 and #2834

**Comment by comintern** [comment]
*Should* `Width` resolve to a function? VBA treats it as a function internally, but externally it's defined as a statement.  The grammar for it appears to be correct:
```
// 5.4.5.7 Width Statement
widthStmt : WIDTH whiteSpace markedFileNumber whiteSpace? COMMA whiteSpace? lineWidth;
lineWidth : expression;
```
The VBE is similarly conflicted - it displays IntelliSense for it...

![untitled](https://cloud.githubusercontent.com/assets/8944005/23824389/749f09e0-063b-11e7-870f-71398c0e09bb.png)

...but it doesn't show an alias for it anywhere in the Object Browser.  It ***doesn't*** exhibit the same behavior for the `Input #` statement or `Write #` statement, or any of the other file handling statements AFAICT.

If RD *does* pick this up as a procedure declaration, it will probably require special handling for it in the resolver. I'm a bit up in the air as to whether we should do that.  If we eventually want to perform code-path analysis on the file handling functions (ref #2375), this might get in the way.

**Comment by ThunderFrame** [comment]
Is it perhaps because the Procedure name is reused as a parameter name? Is that unique to this procedure?

Object Browser and OLEWoo both show it as a standard procedure.

![width](https://cloud.githubusercontent.com/assets/16574009/23826272/569b86a0-06ed-11e7-8abe-5e26c838ff06.png)

```
[helpcontext(0x000f65a4)] 
void _stdcall Width(
	[in] short FileNumber,
	[in] short Width
);
```

**Comment by comintern** [comment]
Yeah, I'm not sure exactly why they have it declared there and not the other file handling statements. I guess the main question. It does seem to be unique to `Width` though - none of the other file statements are declared anywhere.  The `Input*` are all documented as both statements and procedures, but not `Width`.  I wouldn't be surprised if that was added as a bug fix at some point, but who knows.

**Comment by ThunderFrame** [comment]
Oh, but the file-handling statements Input and Input B *are* declared in `_HiddenModule` - they're just the pesky Aliased variety. My current PR fixes their declarations - If only I could work out what is breaking the tests.

```
BSTR _stdcall _B_str_InputB(
	[in] long Number,
	[in] short FileNumber
);
[helpcontext(0x000f735f)] 
VARIANT _stdcall _B_var_InputB(
	[in] long Number,
	[in] short FileNumber
);
[helpcontext(0x000f735f)] 
BSTR _stdcall _B_str_Input(
	[in] long Number,
	[in] short FileNumber
);
[helpcontext(0x000f735f)] 
VARIANT _stdcall _B_var_Input(
	[in] long Number,
	[in] short FileNumber
```

**Comment by bclothier** [comment]
To offer another data point, it turns out that [`Width` can be aliased](https://docs.microsoft.com/en-us/office/vba/language/reference/user-interface-help/widthstatement?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev11.query%3FappId%3DDev11IDEF1%26l%3Den-US%26k%3Dk(vblr6.chm1009060)%3Bk(TargetFrameworkMoniker-Office.Version%3Dv16)%26rd%3Dtrue):

![image](https://user-images.githubusercontent.com/2367644/62942814-b921e700-bd9e-11e9-83fd-ee52d973826c.png)

However, note that I cannot use the `#` prefix in the aliased form:
![image](https://user-images.githubusercontent.com/2367644/62942901-ebcbdf80-bd9e-11e9-8abb-e6a11c02efeb.png)

So the unaliased form _seems_ to be referencing the `Width` statement, and we must prefix if we want to use the function defined in `VBA._HiddenModule.Width`. 

Does that change the analysis?
